### DISCUSSION TITLE
0,50x multiplier on both wind up and wind down

### DISCUSSION THREAD

**Discussion by reo-the-enby** [ppy/osu#21186] [Q&A] [answered]
Shouldn't wind up have >1x multiplier?

**Comment by mk56-spn** [comment, marked as answer]
No. Wind up and wind down simply indicate the direction of the speed change as their names say. Not from where you start and end


For example wind up starting with .5 speed and ending at .55 is ridiculously easier than NM ( and on the other end of the spectrum wind down with 2x to 1.95x is infinitely harder than NM) .

as such for now they both have temporary negative score multipliers that neuter their ability to be leaderboard relevant until a better solution is ready
